### PR TITLE
Show followers

### LINKED ISSUE #38: Make "who follows me" visible
This is an important component for privacy.

### PR DESCRIPTION
**Pull request by wlonk**
Fixes #38.

### CODE REVIEW

File: templates/wheretofindme/follower_list.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block "main" %}
+<h1 class="display-4">Who follows me?</h1>
+<div>
+  {% for follow in object_list %}
+    <div class="card bg-light shadow-sm mb-3">
+      <a href="{% url "user-profile" slug=follow.from_user.username %}">
```
Comment (ojacobson) [inline templates/wheretofindme/follower_list.html]: The same point about showing a pivot table I made on #29 applies here. Being able to quickly scan the list and see _who these people are_ is probably useful.

File: templates/wheretofindme/follower_list.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        <div class="card-body">
+          {{ follow.from_user.username }}
+        </div>
+      </a>
+    </div>
+  {% empty %}
+    No one's following you yet.
```
Comment (ojacobson) [inline templates/wheretofindme/follower_list.html]: `s/ yet//`

File: wheretofindme/views.py [hunk 3]
```diff
@@ -49,6 +49,14 @@ def get_queryset(self):
         return Follow.objects.filter(from_user=self.request.user)
 
 
+class FollowersView(LoginRequiredMixin, ListView):
+    model = Follow
+    template_name = "wheretofindme/follower_list.html"
+
+    def get_queryset(self):
+        return Follow.objects.filter(to_user=self.request.user)
```
Comment (ojacobson) [inline wheretofindme/views.py]: We're gonna want an index on `to_user` to make this go reasonably fast on larger datasets.

Comment (wlonk) [reply wheretofindme/views.py]: Looks like Django automatically adds indexes for ForeignKeys.

File: templates/wheretofindme/_identity.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<span class="badge badge badge-light">
+  {% if identity.looks_like_link %}
```
[the remark this answers is not in the document]
Comment (wlonk) [reply templates/wheretofindme/_identity.html]: Tend to agree. Not enthused about how I wrote it.

**Pull request merged**